.gitlab-ci.yml

```diff
@@ -1,197 +1,152 @@
 stages:
   - test
-  - push-release
```

Comment:
I think this should stay. It notifies upater about new version (not specific build)